### ISSUE TITLE
Can't compile latest version

### ISSUE THREAD

**Issue by NikolayJuly** [SDWebImage/libde265-Xcode#2]
I just added SDWebImageHEIFCoder package to my xcode project.

I see that there was bump version to 1.07 couple month back. But include header here wasn't updated `libde265-Xcode/include/libde265/de265.h`. As result 4 functions which is needed by libheif are missing:
```
LIBDE265_API int de265_get_image_full_range_flag(const struct de265_image*);
LIBDE265_API int de265_get_image_colour_primaries(const struct de265_image*);
LIBDE265_API int de265_get_image_transfer_characteristics(const struct de265_image*);
LIBDE265_API int de265_get_image_matrix_coefficients(const struct de265_image*);
```

Please update header in include folder as well. 

Compilation failed on macOS, on iOS via pods everything works

**Comment by dreampiggy** [comment]
Seems we need to symbolic link to the original header, but not copy from source. Which may cause unsync in the future.

I'll crearte PR to fix this.

**Comment by NikolayJuly** [comment]
Cool, in this case I will close [my one](https://github.com/SDWebImage/libde265-Xcode/pull/3). This was simpler one time fix, rather than symbolik link.

**Comment by dreampiggy** [comment]
Fixed by commit: dfde8066c3751ee2003f1aade063b842a8127375

I'll release 1.0.8 version now.

**Comment by NikolayJuly** [comment]
Thanks, I will close issue then.
